lib/model/imapmessage.php

```diff
@@ -479,6 +479,9 @@ public function getFullMessage($ownMail, $specialRole=null) {
 		$data = $this->getListArray();
 		if ($this->hasHtmlMessage) {
 			$data['hasHtmlBody'] = true;
+			if($specialRole === 'drafts'){
+				$data['body'] = rtrim($this->htmlMessage,"<br><br>");
```

Comment:
the root of this problem. We should debug where those line breaks come from and remove those there instead of patching this here. Does that make sense?